src/assignment.ts

```diff
@@ -1,5 +1,19 @@
+export type AssignmentData = {
+  split_name: string;
+  variant: string;
+  context?: string;
+  unsynced: boolean;
+};
+
+export type AssignmentOptions = {
+  splitName: string;
+  variant: string | null;
+  context?: string;
```

Comment:
i think context is required.

Reply:
We use this class in `TestTrackConfig` and `context` is not passed in there.